packages/iTwinUI-react/src/core/Table/TableRowMemoized.tsx

```diff
@@ -74,18 +74,19 @@ export const TableRow = <T extends Record<string, unknown>>(props: {
     root: intersectionRoot,
   });
 
-  const userRowProps = rowProps?.(row);
+  const userRowProps = rowProps?.(row) ?? {};
+  const { status, ...restUserRowProps } = userRowProps;
   const mergedProps = {
     ...row.getRowProps({ style: { flex: `0 0 auto`, minWidth: '100%' } }),
-    ...userRowProps,
+    ...restUserRowProps,
     ...{
       className: cx(
         'iui-row',
         {
           'iui-selected': row.isSelected,
           'iui-row-expanded': row.isExpanded && subComponent,
           'iui-disabled': isDisabled,
-          [`iui-${userRowProps?.status}`]: userRowProps?.status,
+          [`iui-${status}`]: !!status,
```

Comment:
If `status` is not in `userRowProps`, would line 89 become `iui-undefined: false`? But I guess it wouldn't matter because `!!status` would be false and so `iui-undefined` would not be displayed. Have I understood correctly?

Reply:
`iui-status` will not be added to classNames unless `!!status` is true.